src/crayon.jl

```diff
@@ -103,7 +103,8 @@ Base.inv(c::Crayon) = Crayon(inv(c.fg), inv(c.bg), ANSIStyle(), # no point takin
                              inv(c.blink), inv(c.negative), inv(c.conceal), inv(c.strikethrough))
 
 function Base.print(io::IO, x::Crayon)
-    if anyactive(x) && (Base.have_color || _force_color())
```

Comment:
I think we can drop the `Base.have_color` completely since that determines if the `io` get a `:color` field anyway?

Reply:
Good! I though that `Base.have_color` had some more general meaning, like Julia was compiled without color support or something. I will change now.

Reply:
Done!